Formula/s/sigsum.rb

```diff
@@ -0,0 +1,28 @@
+class Sigsum < Formula
+  desc "Key transparency toolkit"
+  homepage "https://sigsum.org"
+  url "https://git.glasklar.is/sigsum/core/sigsum-go/-/archive/v0.9.1/sigsum-go-v0.9.1.tar.bz2"
+  sha256 "68f3a66af6727fa1b44d7dd2429748b23b8767547750113318494c58d7b04ae8"
+  license "BSD-2-Clause"
+
+  depends_on "go" => :build
+
+  def install
+    %w[
+      sigsum-key
+      sigsum-monitor
+      sigsum-submit
+      sigsum-token
+      sigsum-verify
+      sigsum-witness
+    ].each do |cmd|
+      system "go", "build", *std_go_args(output: bin/cmd, ldflags: "-s -w"), "./cmd/#{cmd}"
+    end
+  end
+
+  test do
+    system bin/"sigsum-key", "gen", "-o", "key-file"
+    pipe_output("#{bin}/sigsum-key sign -k key-file -o signature", "#{bin}/sigsum-key")
+    pipe_output("#{bin}/sigsum-key verify -k key-file.pub -s signature", "#{bin}/sigsum-key")
```

Comment:
can we also verify the output dump?

```suggestion
    pipe_output("#{bin}/sigsum-key sign -k key-file -o signature", bin/"sigsum-key")
    pipe_output("#{bin}/sigsum-key verify -k key-file.pub -s signature", bin/"sigsum-key")
```

Reply:
I'm not sure what you mean by verifying the output dump? This test case verifies the signature against the original file, not sure what else it should be verifying?